sign-verify.md

```diff
@@ -7,9 +7,8 @@ Utilize the following steps to sign a container that has been published to an OC
 The `base_hostname` can be obtained from
 
 ```shell
-oc get dnsrecords -o yaml -n openshift-ingress-operator | grep dnsName
-# omit the '*.'
-export BASE_HOSTNAME=apps.something.something.openshiftapps.com
+oc get dns cluster -o jsonpath='{ .spec.baseDomain }'
+export BASE_HOSTNAME=apps.BASE_DOMAIN
```

Comment:
Do we want to align the variable `OPENSHIFT_APPS_SUBDOMAIN`. And then just use a single `OPENSHIFT_APPS_SUBDOMAIN=apps.$(oc get dns cluster -o jsonpath='{ .spec.baseDomain }')`>

Reply:
yes, I was thinking that as well - will push that change

Reply:
updated - we'll want to also update the env variable in the cosign Dockerfile, I'll add that to the list of follow up TODOs

Reply:
I updated in default values.yaml from `cosign.BASE_DOMAIN` to `cosign.appsSubdomain`